assets/dev/js/editor/controls/media.js

```diff
@@ -89,9 +89,9 @@ ControlMediaItemView = ControlMultipleBaseItemView.extend( {
 		// Set current doc id to attach uploaded images.
 		wp.media.view.settings.post.id = elementor.config.document.id;
 		this.frame = wp.media( {
-			button: {
-				text: __( 'Insert Media', 'elementor' ),
-			},
+			frame: 'post',
```

Comment:
Please add before/after screenshots.

Reply:
It looks the same.

Reply:
Even after removing unwanted elements from modal?

Reply:
When you said "before/after", you meant specifically those lines you quoted or everything?

Reply:
Everything